gtk4/src/subclass/widget.rs

```diff
@@ -123,3 +126,127 @@ unsafe extern "C" fn widget_direction_changed<T: ObjectSubclass>(
 
     imp.direction_changed(&wrap, dirwrap)
 }
+
+pub unsafe trait WidgetClassSubclassExt
+where
+    Self: ObjectSubclass + WidgetImpl,
+{
+    fn set_template_bytes(
+        klass: &mut subclass::simple::ClassStruct<Self>,
```

Comment:
```suggestion
        klass: &mut glib::Class<Self::Type>,
```

or

```suggestion
        klass: &mut glib::Class<<Self as ObjectSubclass>::Type>,
```

Here and in the other functions.

Reply:
No, actually this should be:

```suggestion
        klass: &mut <Self as ObjectSubclass>::Class,
```